MapboxSearch/autofill/src/main/java/com/mapbox/search/autofill/AddressAutofillResult.kt

```diff
@@ -46,7 +58,9 @@ public class AddressAutofillResult internal constructor(
      * @suppress
      */
     override fun hashCode(): Int {
-        var result = suggestion.hashCode()
+        var result = id.hashCode()
+        result = 31 * result + mapboxId.hashCode()
```

Comment:
Does this hash function still make sense with the introduction the mapboxId?
Should we consider simply using the mapboxId when one is available?

Reply:
It must be some Java/Kotlin thing, I don't think we use it anywhere directly. Also, `mapboxId` may be empty.